src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmOutcome.cshtml

```diff
@@ -232,10 +256,31 @@
                                           id="OptionRejectedText"
                                           name="OptionRejectedText"
                                           rows="5"
-                                          aria-describedby="OptionFailedText-info">@Model.OptionRejectedText</textarea>
+                                          aria-describedby="OptionRejectedText-info OptionRejectedText-hint">@Model.OptionRejectedText</textarea>
                                 <span id="OptionRejectedText-info" class="govuk-hint govuk-character-count__message" aria-live="polite"></span>
                             </div>
                         </div>
+                        <div class="govuk-character-count" data-module="govuk-character-count" data-maxwords="500">
```

Comment:
150 words here?

Reply:
Rejected -> External comments. AC5 and the content doc both say 500 words.